apps/vr-tests-react-components/src/stories/Menu/Menu.stories.tsx

```diff
@@ -15,8 +15,10 @@ import {
   MenuSplitGroup,
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 52 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28119&commitId=196f45f2a77f97ee4bf94e202cb7ab2ea0894f3d&env=prod)

✅ There was 2 screenshots added, 27 screenshots removed, 1894 screenshots unchanged, 0 screenshots with different dimensions and 23 screenshots with visible difference.


  <details><summary><b>unknown</b> 52 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - basic.default - RTL.chromium.png|1861| Changed|
|Menu Converged - basic.default.chromium.png|1594| Changed|
|Menu Converged - groups.default - Dark Mode.chromium.png|40538| Changed|
|Menu Converged - groups.default - High Contrast.chromium.png|49307| Changed|
|Menu Converged - groups.default - RTL.chromium.png|3465| Changed|
|Menu Converged - groups.default.chromium.png|3163| Changed|
|Menu Converged - icon slotted content.default - RTL.chromium.png|2271| Changed|
|Menu Converged - icon slotted content.default.chromium.png|2011| Changed|
|Menu Converged - nested submenus.default - RTL.chromium.png|2476| Changed|
|Menu Converged - nested submenus.default.chromium.png|2207| Changed|
|Menu Converged - secondary content.default - RTL.chromium.png|2137| Changed|
|Menu Converged - secondary content.default.chromium.png|1869| Changed|
|Menu Converged - selection groups.default - RTL.chromium.png|3557| Changed|
|Menu Converged - selection groups.default.chromium.png|3251| Changed|
|Menu Converged - selection.checkbox - Dark Mode.chromium.png|36523| Changed|
|Menu Converged - selection.checkbox - High Contrast.chromium.png|43674| Changed|
|Menu Converged - selection.checkbox - RTL.chromium.png|1861| Changed|
|Menu Converged - selection.checkbox.chromium.png|1594| Changed|
|Menu Converged - split item.default - RTL.chromium.png|2308| Changed|
|Menu Converged - split item.default.chromium.png|2038| Changed|
|Menu Converged - submenuIndicator slotted content.default - RTL.chromium.png|1384| Changed|
|Menu Converged - submenuIndicator slotted content.default.chromium.png|1129| Changed|
|Menu nested within a MenuTrigger.default.chromium.png|3446| Changed|
|Menu.Nested Submenus Small Viewport Flipped.chromium.png|0| Added|
|Menu.Nested Submenus Small Viewport Stacked.chromium.png|0| Added|
|Menu Converged - basic.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - basic.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - icon slotted content.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - icon slotted content.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - nested submenus.default - RTL.all open.chromium.png|0| Removed|
|Menu Converged - nested submenus.default.all open.chromium.png|0| Removed|
|Menu Converged - secondary content.default - RTL.hover menuitem.chromium.png|0| Removed|
|Menu Converged - secondary content.default.hover menuitem.chromium.png|0| Removed|
|Menu Converged - selection groups.default - RTL.selected.chromium.png|0| Removed|
|Menu Converged - selection groups.default.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - Dark Mode.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - High Contrast.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox - RTL.selected.chromium.png|0| Removed|
|Menu Converged - selection.checkbox.selected.chromium.png|0| Removed|
|Menu Converged - split item.default - RTL.submenu open.chromium.png|0| Removed|
|Menu Converged - split item.default.submenu open.chromium.png|0| Removed|
|Menu Converged - submenuIndicator slotted content.default - RTL.submenus open.chromium.png|0| Removed|
|Menu Converged - submenuIndicator slotted content.default.submenus open.chromium.png|0| Removed|
|Menu nested within a MenuTrigger.default.submenu open.chromium.png|0| Removed|
|Menu.Nested Submenus Small Viewport Flipped.default.chromium.png|0| Removed|
|Menu.Nested Submenus Small Viewport Flipped.nested menu.chromium.png|0| Removed|
|Menu.Nested Submenus Small Viewport Stacked.default.chromium.png|0| Removed|
|Menu.Nested Submenus Small Viewport Stacked.nested menu.chromium.png|0| Removed|
|Popover Converged.avoid scrolling.PopoverSurface focused.chromium.png|0| Removed|
|Popover Converged.positioning - High Contrast.positioned popovers.chromium.png|0| Removed|
|Popover Converged.positioning - RTL.positioned popovers.chromium.png|0| Removed|
|Popover Converged.positioning.positioned popovers.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>